tests/test_stateupdate_authority.c

```
@@ -0,0 +1,138 @@
+
+    Sleep(250);
+    test_client_drain(&a, 300);
+    test_client_drain(&b, 300);
+
+    /* Send an owner-style movement update from A. */
+    u8 su[128];
+    int su_len = build_owner_stateupdate(bc_make_ship_id(0),
+                                         123.0f, -77.0f, 19.0f,
+                                         8.0f,
+                                         su, sizeof(su));
+    ASSERT(su_len > 0);
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)